_posts/2023-06-12-may-rubygems-updates.md

```diff
@@ -0,0 +1,65 @@
+---
+title: May 2023 RubyGems Updates
+layout: post
+author: Gift Egwuenu
+author_email: laurandidi21@gmail.com
+---
+
+Welcome to the RubyGems monthly update! As part of our efforts at Ruby Central, we publish a recap of the work that we’ve done the previous month. Read on to find out what updates were made to RubyGems and RubyGems.org in May.
+
+## RubyGems News
+
+This month in RubyGems, we released RubyGems [3.4.13](https://github.com/rubygems/rubygems/blob/master/CHANGELOG.md#3413--2023-05-09) and Bundler [2.4.13](https://github.com/rubygems/rubygems/blob/master/bundler/CHANGELOG.md#2413-may-9-2023).
+
+The following improvements and fixes are included in these releases (see the changelog for more information):
```

Comment:
Since the above release happened very early in the month, instead of the below list (from which most improvements have not actually been released), I'd say something like the following (also adding a few more improvements):

> In addition to the improvements listed in the above changelogs, other changes that landed in our repo during May were:

> * Ongoing unification of RubyGems and Bundler codebases - [#6691](https://github.com/rubygems/rubygems/pull/6691), [#6716](https://github.com/rubygems/rubygems/pull/6716).
> * Fix API key loading if RubyGems host is set to development - [#6683](https://github.com/rubygems/rubygems/pull/6683).
> * Show an error when trying to update bundler in frozen mode - [#6684](https://github.com/rubygems/rubygems/pull/6684).
> * Make frozen setting take precedence over deployment setting - [#6685](https://github.com/rubygems/rubygems/pull/6685).
> * Fix deployment vs path precedence - [#6703](https://github.com/rubygems/rubygems/pull/6703).

Reply:
Thanks, @deivid-rodriguez!  I'll go ahead and update the post.